ddtrace/contrib/internal/asgi/middleware.py

```diff
@@ -29,14 +39,29 @@
 
 log = get_logger(__name__)
 
+if os.getenv("DD_ASGI_TRACE_WEBSOCKET") is not None:
+    log.warning(
+        "DD_ASGI_TRACE_WEBSOCKET is deprecated and will be removed in a future version. "
+        "Use DD_TRACE_WEBSOCKET_MESSAGES_ENABLED instead."
+    )
+
 config._add(
     "asgi",
     dict(
         service_name=config._get_service(default="asgi"),
         request_span_name="asgi.request",
         distributed_tracing=True,
+        trace_asgi_websocket_messages=asbool(
+            os.getenv("DD_TRACE_WEBSOCKET_MESSAGES_ENABLED", default=os.getenv("DD_ASGI_TRACE_WEBSOCKET", False))
```

Comment:
These should use [`get_config`](https://github.com/DataDog/dd-trace-py/blob/819c74c98e0497f90ee1bb6edc589b4a00f4c195/ddtrace/internal/telemetry/__init__.py#L42) to conform with precedence rules for configuration sources. Apologies for not catching this on the first pass.